content/en/docs/concepts/verification-by-reference.md

```diff
@@ -354,13 +347,13 @@ The reverse lookup index could be expanded as follows:
 
 The user still references `registry.contoso.com/web:a2b2`, while other parts of the system that adhere to Notary would know to query the registry for the additional verification information.
 
-An nv2 client queries for all indexes with a `"config.mediaType"` of `"application/vnd.cncf.notary.verification.config.v1+json"` It would read all the verification objects and decide how to proceed.
+An notation client queries for all indexes with a `"config.mediaType"` of `"application/vnd.cncf.notary.verification.config.v1+json"` It would read all the verification objects and decide how to proceed.
```

Comment:
Suggest remove this sentence for now. It is not correct. We can update later to put more accurate description here.